adapters/utils/runAdapter.ts

```diff
@@ -143,20 +143,22 @@ async function _runAdapter({
   }
 
   let breakdownByToken: any = {}
+  let breakdownData: any = {}
```

Comment:
replace breakdown with breakdownByLabel to avoid confusion?

We used to have a BreakdownAdapter, I have most code related to that, but still